aea/cli/search.py

```diff
@@ -41,11 +42,28 @@ def search(ctx: Context, registry):
     if registry:
         ctx.set_config("is_registry", True)
     else:
-        registry = os.path.join(AEA_DIR, DEFAULT_REGISTRY_PATH)
+        registry = os.path.join(ctx.cwd, DEFAULT_REGISTRY_PATH)
         ctx.set_config("registry", registry)
         logger.debug("Using registry {}".format(registry))
 
 
+def _is_invalid_item(name, dir_path, config_path):
+    """Return true if this protocol, connection or skill should not be returned in the list."""
+    return ".py" in name or "__" in name or name == "scaffold" or os.path.isfile(dir_path) or not os.path.isfile(config_path)
```

Comment:
We should also exclude `.DS_Store`, so really any item starting with `.` as we don't consider that valid naming of an item

Reply:
.DS_Store is already invalid because it doesn't contains a config file.